fluid-project/handbook.floeproject.org
# fix: remove subsection links from breadcrumb

## Description

This update disables links within the breadcrumb for entries that are deemed "subsections", i.e. the accordion controls on either the Approaches or Techniques pages.

## Steps to test

1. Go to the Approaches page
2. Click on the "Inclusive Authoring" subsection heading to expand the accordion control
3. Click on "Metadata" to go to the Metadata page

**Expected behavior:** The breadcrumb list on the Metadata page should have an entry for "Inclusive Authoring", but it should not be a link any more.

## Additional information

N/A

## Related issues

Resolves #65
